install.go

```diff
@@ -52,6 +52,10 @@ func (c *Client) installReleaseFromTarball(ctx context.Context, chartPath, names
 }
 
 func (options InstallOptions) configure(action *action.Install, namespace string) {
+	// Disabling OpenAPI Validation which introduced from Helm 3,
+	// since this feature restricted the chart upgrade that we had it from Helm 2.
```

Comment:
Thanks. I will put same comment as install.go!

Reply:
To my surprise, update options do not include `DisableOpenAPIValidation` at all. I would look into it furthermore why and how to disable when it comes to upgrade releases.

Reply:
You're right. I was looking at master but this was added in 3.2.0 and we're using 3.1.2.

Sorry to be a pain but could you update to 3.1.3? As it has a security fix. https://github.com/helm/helm/tree/v3.1.3

I think we should hold off upgrading to 3.2.1 because it uses K8s 1.18 instead of 1.17. WDYT?